app/forms/idv/consent_form.rb

```diff
@@ -2,17 +2,17 @@ module Idv
   class ConsentForm
     include ActiveModel::Model
 
-    validates :ial2_consent_given?,
+    validates :idv_consent_given?,
               acceptance: { message: proc { I18n.t('errors.doc_auth.consent_form') } }
 
     def submit(params)
-      @ial2_consent_given = params[:ial2_consent_given] == '1'
+      @idv_consent_given = params[:idv_consent_given] == '1' || params[:ial2_consent_given] == '1'
```

Comment:
Personally I'd go for one of these
```suggestion
      @idv_consent_given = (params[:idv_consent_given] || params[:ial2_consent_given]) == '1'
```